src/components/themed/WiredProgressBar.js

```diff
@@ -85,3 +75,20 @@ const getStyles = cacheStyles((theme: Theme) => ({
     zIndex: 100
   }
 }))
+
+export const WiredProgressBar = connect(
+  (state: RootState): StateProps => {
+    const walletsForProgress = state.ui.wallets.walletLoadingProgress
+    const walletIds = Object.keys(walletsForProgress)
+    if (walletIds.length === 0) return { progress: 0 }
+
+    let sum = 0
+    for (const walletId of walletIds) {
+      sum += walletsForProgress[walletId]
+    }
+    let ratio = sum / walletIds.length
+    if (ratio > 0.9999) ratio = 1
```

Comment:
should it be the same value as was before? `0.9999`

Reply:
Oops, fixed.